cocotb/share/makefiles/Makefile.pylib.Msys

```diff
@@ -54,5 +54,30 @@ empty :=
 PYTHON_VERSION=$(subst $(period),$(empty),$(LOCAL_PYTHON_VERSION))
 PYTHON_INCLUDEDIR := $(shell $(PYTHON_BIN) -c 'import distutils.sysconfig; print( distutils.sysconfig.get_python_inc() )')
 
-PYLIBS = -lPython$(PYTHON_VERSION)
-PYTHON_DYN_LIB = Python$(PYTHON_VERSION).dll
+SYSCONFIG_LDLIBRARY = $(shell $(PYTHON_BIN) -c 'from distutils import sysconfig; print( sysconfig.get_config_var("LDLIBRARY") )')
+
+ifeq ($(SYSCONFIG_LDLIBRARY),None)
+    PYLIBS := -lPython$(PYTHON_VERSION)
+else
+    PYLIBS:=$(shell $(PYTHON_BIN) -c 'from distutils import sysconfig; import os; print("-l"+os.path.splitext(sysconfig.get_config_var("LDLIBRARY"))[0][3:])')
+endif
+
+# Some simulator accept only paths in form of "c:/../.."
+ifneq ($(VERILOG_SOURCES),)
+    VERILOG_SOURCES := $(shell cygpath -m $(VERILOG_SOURCES))
+endif
+ifneq ($(VHDL_SOURCES),)
+    VHDL_SOURCES := $(shell cygpath -m $(VHDL_SOURCES))
+endif
+
+PWD ?= $(shell pwd)
+
+# sys.path in form "/c/../:/c/.."
+PYTHON_SYS_PATH := $(shell $(PYTHON_BIN) -c "import sys, os; print(':'.join(['/'+dir.replace('\\\ ',' ').replace(os.sep,'/').replace(':','') for dir in sys.path]))")
```

Comment:
':' is better spelt `os.sep`

Reply:
This actully does not work. `os.pathsep` is `;` on windows and for whatever reason it does not like it.

Reply:
Huh, I wasnt partying attention here, : is correct.